launcher_scripts/conf/training/gpt3/400m_improved.yaml

```diff
@@ -126,14 +126,16 @@ model:
   gc_interval: 100 # Interval of the host memory garbage collection
 
   ## Transformer Engine
-  transformer_engine: False
+  # To use fp8, change only `fp8` to `True`.
+  # The rest of fp8 knobs are set for the fp8 training mode, which are ignored in non-fp8 training
+  transformer_engine: True
```

Comment:
all improved configs should have transformer_engine=False as they don't yet work with TE due to lack of rope and swiglu support

Reply:
Done